Week_01_Basics/Assignment/Portfolio.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<link rel="apple-touch-icon" sizes="57x57" href="/favicons/apple-touch-icon-57x57.png">
```

Comment:
```<link>``` ```<meta>``` tags belong in the ```<head>``` tag.  Nothing should exist outside of the head and body tags apart from html and !doctype